vagrant/hieradata/common.yaml

```diff
@@ -0,0 +1,29 @@
+# Infrastructure
+cmf::infrastructure::packages::installs:
+  - vim
+  - git
+  - wget
+  - bash-completion
+  - ruby-dev
+  - software-properties-common
+  - python-software-properties
+
+# Application
+cmf::application::php::version: "5.6" # Available are "5.4", "5.5", "5.6" and "7.0"
+
+# Storage
+cmf::infrastructure::storage::mysql: true
+cmf::infrastructure::storage::jack: false
+
+# MySQL
+cmf::infrastructure::phpcr::mysql_wrapper::root_password: root
```

Comment:
in the end everything which depends on jackrabbit or mysql should be uncommented, so the user avoids leaving stuff commented which is actually not needed